weave/execute.py

```diff
@@ -223,7 +223,7 @@ def execute_nodes(nodes, no_cache=False) -> value_or_error.ValueOrErrors[typing.
                         # TODO: Get errors from here
                         stats = execute_forward(fg, no_cache=no_cache)
                         summary = stats.op_summary()
-                        logging.info("Execution summary\n%s" % pprint.pformat(summary))
+                        logging.info("Execution summary", summary)
```

Comment:
I wonder if there's a way to keep this structure for the JSON / DataDog logs...  I think at a minimum we should only get rid of the `pformat` when we're not in development mode.  Printing as is won't be readable in dev.

Reply:
@vanpelt @dannygoldstein and I are actually pairing on this right now, and we're walking back this part of the change -- it wasn't necessary after all